modules/data-streams/src/test/java/org/elasticsearch/datastreams/action/GetDataStreamsResponseTests.java

```diff
@@ -43,22 +61,221 @@ protected Response mutateInstance(Response instance) {
         return new Response(instance.getDataStreams().stream().map(this::mutateInstance).toList());
     }
 
+    @SuppressWarnings("unchecked")
+    public void testResponseILMandDataStreamLifecycleRepresentation() throws Exception {
```

Comment:
Nit: I think the correct camel case here is: `testResponseIlmAndDataStreamLifecycleRepresentation `

Reply:
Ah ++. Fixed, thank you